style.css

```diff
@@ -0,0 +1,473 @@
+    margin: 0 35px 0 0;
+}
+.nav__link {
+    color: #fff;
+    text-decoration: none;
+    font-size: 22px;
+    line-height: 26px;
+    font-family: 'Museo';
+    border-bottom: 1px solid transparent;
+    transition: .3s ease;
+}
+.nav__link:hover {
+    border-bottom: 1px solid #ffffff;
+    opacity: 0.8;
+}
+.nav__toggle {
+    display: none;
+}
+.nav__burder{
+    display: none;
+}
+.bali {
+    display: flex;
+    flex-direction: column;
+    margin-left: 60px;
+    padding-left: calc(100vw - 1780px)
+}
+.bali-wrapper {
+    position: relative;
+
+}
+.bali__title {
+    display: flex;
+    font-family: 'PTSerif';
+    font-style: normal;
+    margin: 0;
+    font-weight: bold;
+    font-size: 280px;
+    color: #f8eee4;
+    opacity: 0.4;
+}
+.bali__subtitle {
+    font-family: 'PTSerif';
+    font-style: normal;
+    font-weight: bold;
+    font-size: 98px;
+    margin: 0;
+    color: #F8EEE4;
+    position: absolute;
+    left: 83px;
+    bottom: 11px;
+}
+.bali__text {
+    font-family: 'Museo';
+    font-style: normal;
+    font-weight: normal;
+    font-size: 22px;
+    line-height: 26px;
+    color: #F8EEE4;
+    margin: 9px 0 24px 10px;
+    max-width: 551px;
+    padding-left: 10px;
+}
+.picture {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding-top: 115px;
+}
+.picture__pic {
+    padding-bottom: 3px;
+}
+.picture__pic:not(:last-child) {
+    margin: 0 32px 0 0;
+}
+.picture__pic_active {
+    padding-bottom: 0;
+}
+.picture__pic_cut{
+    margin-left: 1px;
+    margin-right: 0;
+}
+.picture__pic_tablet{
+    display: none;
+}
+.bali__btn {
+    width: 292px;
+    height: 61px;
+    background: #D4BEA3;
+    border-radius: 48px;
+    border: none;
+    font-weight: bold;
+    font-size: 22px;
+    line-height: 26px;
+    color: #ffffff;
+    cursor: pointer;
+    transition: .5s;
+    margin-left: 23px;
+    margin-top: 14px;
+}
+.bali__btn:hover {
+    background: none;
+    border: 1px solid #D4BEA3;
+}
+#cut{
+    margin-right: 0;
+}
+
+@media screen and (min-width: 1920px){
+    .bali {
+        display: flex;
+        flex-direction: column;
+        margin-left: 60px;
+        padding-left: 200px;
+    }
+    
+}
+@media screen and (max-width: 1600px){
+    header > img{
+        width: 104px;
+        height: 39px
+    }
+    .header{
+        padding: 63px 75px 0 142px;
+        margin: 0 0 90px 0;
+    }
+    .nav__link:not(:last-child) {
+        margin: 0 24px 0 0;
+    }
+    .nav__link{
+        font-size: 16px;
+        line-height: 19px;
+    }
+    .picture__pic:not(:last-child) {
+        margin: 0 23px 0 0;
+    }
+    .picture__pic{
+        width: 203px;
+        height: 270px;
+    }
+    .picture__pic_active{
+        width: 226px;
+        height: 300px;
+    }
+    aside > .picture__pic_cut{
+        width: 148px;
+        margin: 0;
+    }
+    .bali__title{
+        font-size: 200px;
+        line-height: 265px;
+        margin-top: 1px;
+        left: 59px;

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Indonesia/pullRequest?prid=5674203)